t/t1014-mf-priority-dne.t

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+test_description='Test cancelling active jobs with a late user/bank info load'
+
+. `dirname $0`/sharness.sh
+MULTI_FACTOR_PRIORITY=${FLUX_BUILD_DIR}/src/plugins/.libs/mf_priority.so
+
+export TEST_UNDER_FLUX_SCHED_SIMPLE_MODE="limited=1"
+test_under_flux 1 job
+
+flux setattr log-stderr-level 1
+
+test_expect_success 'load multi-factor priority plugin' '
+	flux jobtap load -r .priority-default ${MULTI_FACTOR_PRIORITY}
+'
+
+test_expect_success 'check that mf_priority plugin is loaded' '
+	flux jobtap list | grep mf_priority
+'
+
+test_expect_success 'submit job #1 with no user/bank info loaded to plugin' '
+	jobid1=$(flux mini submit hostname)
+'
+
+test_expect_success 'check that job #1 is in state PRIORITY' '
+	test $(flux jobs -no {state} ${jobid1}) = PRIORITY
+'
+
+test_expect_success 'send the user/bank information to the plugin without reprioritizing all active jobs' '
+	cat <<-EOF >fake_payload.py
+	import flux
+	import pwd
+	import getpass
+	import json
+
+	username = getpass.getuser()
+	userid = pwd.getpwnam(username).pw_uid
+	# create an array of JSON payloads
+	bulk_update_data = {
+		"data" : [
+			{
+				"userid": userid,
+				"bank": "account3",
+				"def_bank": "account3",
+				"fairshare": 0.45321,
+				"max_running_jobs": 10,
+				"max_active_jobs": 12,
+				"queues": "standby,special"
+			}
+		]
+	}
+	flux.Flux().rpc("job-manager.mf_priority.rec_update", json.dumps(bulk_update_data)).get()
+	bulk_queue_data = {
+		"data" : [
+			{
+				"queue": "default",
+				"priority": 0,
+				"min_nodes_per_job": 0,
+				"max_nodes_per_job": 5,
+				"max_time_per_job": 64000
+			}
+		]
+	}
+	flux.Flux().rpc("job-manager.mf_priority.rec_q_update", json.dumps(bulk_queue_data)).get()
+	EOF
+	flux python fake_payload.py
+'
+
+test_expect_success 'submit job #2 which should run before job #1' '
+	jobid2=$(flux mini submit hostname) &&
+	flux job wait-event $jobid2 clean
```

Comment:
In case something goes wrong and `jobid2` doesn't get a `clean` event (e.g. a bug is introduced and the job is blocked from running somehow), it is useful to provide the `-t, --timeout` option to `flux job wait-event` in tests:
```suggestion
	jobid2=$(flux mini submit hostname) &&
	flux job wait-event -t 15 $jobid2 clean
```

This will cause the test to fail instead of hanging indefinitely.